test_project/adam.py

```diff
@@ -7,6 +7,9 @@
 def constant_number():
     return 42
 
+def constant_boolean():
```

Comment:
I think we should have `constant_true()` and `constant_false()` functions (with associated tests) so that we can verify behavior in both directions.